store/mockstore/tikv.go

```diff
@@ -80,6 +81,13 @@ func WithPath(path string) MockTiKVStoreOption {
 	}
 }
 
+// WithTxnLocalLatches enable or disble txnLocalLatches.
+func WithTxnLocalLatches(enable bool) MockTiKVStoreOption {
```

Comment:
Why not call it SetTxnLocalLatches?

Reply:
We already names previous function in format `Withxxxx`:

```
// WithHijackClient hijacks KV client's behavior, makes it easy to simulate the network
// problem between TiDB and TiKV.
func WithHijackClient(wrap func(Client) Client) MockTiKVStoreOption {
	return func(c *mockOptions) {
		c.clientHijack = wrap
	}
}

// WithCluster provides the customized cluster.
func WithCluster(cluster *mocktikv.Cluster) MockTiKVStoreOption {
	return func(c *mockOptions) {
		c.cluster = cluster
	}
}

// WithMVCCStore provides the customized mvcc store.
func WithMVCCStore(store mocktikv.MVCCStore) MockTiKVStoreOption {
	return func(c *mockOptions) {
		c.mvccStore = store
	}
}
```